### ISSUE TITLE
Etherbase account password wont work- all other accounts including 2nd original account does work.. whats up?

### ISSUE THREAD

**Issue by pwoody8** [ethereum/mist#419]
Node type: geth/eth
OS: linux/windows/osx
Network type: main/test
Mist version: 0.0.0

Please check the already existing issues, before creating an issue. We would like to keep duplicates at a minumim.
Read the https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide first, where we summarized a few of the known issues.

Add the following to issue, when possible:

- Screenshots
- Check the console, of Mist (CTRL/CMD + ALT + i) and take a screenshot.
- Log files (osx: ~/Library/Application Support/Mist/node.log, windows: %APPDATA%/Roaming/Mist/node.log, linux: ~/.config/Mist/node.log)

**Comment by pwoody8** [comment]
I created a Mist wallet and set up account. Default was an etherbase and second account. I fooled around with sending funds to etherbase acct then sent to 2nd account and worked. Messed around with a contract and when I tried to launch contract it said wrong password... then tried to transfer eth between accounts and also indicated wrong password.... created new account (2 more addresses) and was able to send from second account to the new 3rd and 4th accts but still not from first etherbase account. (hope thats clear). It shouldnt be password as it works on acct 2 and worked prior.. any ideas?
